scripts/prog15.rb

```diff
@@ -0,0 +1,18 @@
+def prime(n)
+    (2..n).step(1).each do |value|
+        c=0
+        (2..value/2).each do |e|
+            if value%e==0
+                c+=1
```

Comment:
Instead of integers(0, 1), we can always use boolean values(true, false). While reading a code they look more meaningful.